src/index.ts

```diff
@@ -95,10 +95,30 @@ module.exports = class ServerlessPlugin {
 
   private async afterPackageFunction() {
     const config = getConfig(this.serverless.service);
+    const forwarderArn: string | undefined = config.forwarderArn;
+    const forwarder: string | undefined = config.forwarder;
+
+    let functionArn;
```

Comment:
I think the name of this variable could potentially be a bit confusing because it seems to be the ARN of the Forwarder, not of the function being deployed by the plugin.

Reply:
I am a bit confused with the comment, "let functionArn" is supposed to represent the Datadog Forwarder Arn that is passed into the addCloudWatchForwarderSubscriptions function, I changed the "functionArn" variable to "datadogForwarderArn" to be more explicit.